test/IDE_tests.jl

```diff
@@ -204,3 +204,56 @@ w_real  = [1/x^2 for x in xs]
 # plot!(xs,u_predict)
 # plot(xs,w_real)
 # plot!(xs,w_predict)
+
+println("Improper (a, Inf) Integral Test")
+
+@parameters x
+@variables u(..)
+I = Integral(x in ClosedInterval(1, Inf))
+eqs_ = I(u(x)) ~ 1
```

Comment:
it is not an integral equation and this system of equations is not fully defined, that is, it has a subset of solutions.
Like this example:
![image](https://user-images.githubusercontent.com/12683885/145413544-fb5e94cf-0809-4697-b172-2fd609050449.png)

Reply:
But it did approximate correctly because of the boundary conditions. But I can use another system instead of this. 

If you have any you want me to test, let me know. Otherwise I’ll pick one.